retest1/task3/main.c

```diff
@@ -0,0 +1,37 @@
+#include <stdio.h>
+#include <string.h>
+
+#define CHARS_AMOUNT 256
+
+int* charsCounting(char* filename, char* charCountingArray)
+{
+    FILE* file = fopen(filename, "r");
+    if (file == NULL)
+    {
+        return -1;
+    }
+    int size = 0;
+    char buffer[100] = { 0 };
```

Comment:
```suggestion
    char buffer[100] = { `\0` };
```
Это то же самое, но зато символ, а не int